README.md

```diff
@@ -66,15 +66,15 @@ $ sudo chmod +x /usr/local/bin/dbmate
 
 **Docker**
 
-Docker images are published to both Docker Hub ([`amacneil/dbmate`](https://hub.docker.com/r/amacneil/dbmate)) and Github Container Registry ([`ghcr.io/amacneil/dbmate`](https://ghcr.io/amacneil/dbmate)).
+Docker images are published to both Docker Hub ([`amacneil/dbmate`]) and Github Container Registry ([`ghcr.io/amacneil/dbmate`]).
 
-Remember to set `--network=host` or see [this comment](https://github.com/amacneil/dbmate/issues/128#issuecomment-615924611) for more tips on using dbmate with docker networking):
```

Comment:
I've never seen this trick of moving the links to the bottom of the markdown file, but I'd prefer to keep things the way they are. Otherwise, it's hard to know whether a link is missing, or what text like "this comment" are pointing to. If there is other readme formatting cleanup, let's do that in a separate PR so it's easy to review.

Reply:
Can I suggest using the named link trick for the alternatives table, as it makes the table _smaller_ to edit? With the in-line links and `:white_check_mark:` &c. emoji, the table wraps all the way down to 5pt, at which point the rest of the text is unreadable.

Other options exist for the formatting of the alternatives:

1. Pivot the table so that the _features_ are columns and the alternatives are the rows.
2. Change it from a table to a list.

The first (pivoting) is _probably_ the easier to do and will support adding other types pretty easily. I’m not sure how many columns would be included.

Reply:
Also, with respect to this particular one, it probably should be a better reference than `this comment`, Say `[this comment][128-comment615924611]`, which would then be `[128-comment615924611]: (https://github.com/amacneil/dbmate/issues/128#issuecomment-615924611)`.

I use this extensively in most of the files that I maintain for projects, like [mime-types-data/History.md](https://github.com/mime-types/mime-types-data/blob/main/History.md).

Reply:
More generally it's better if we can separate reformatting PRs from PRs that actually change the content. So if we want to explore reformatting the comparison table I think we should do that _after_ a very simple PR to "add sqitch".

Reply:
I think that what I’m saying is that I could not "add sqitch" without reformatting the table at least minimally. It’s too large because of the way that it has been built so far.

Reply:
That's fine - if you use vscode, prettier should reformat it for you. It's easy enough to view the diff without whitespace to see the changes.

I still prefer not moving all the links to the bottom of the file, because it makes it much harder to see if you have created a broken link (e.g. update the text and forget to update the link).